.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: ~1.20
+        go-version: ~1.21
```

Comment:
Nit: It would be nice to update this to use the version in the `go.mod` file. Can do so with:
```yaml
uses: actions/setup-go@v4
with:
  go-version-file: "go.mod"
```

It would have to be moved to after the checkout action though.